charts/incubator/lldap/values.yaml

```diff
@@ -0,0 +1,57 @@
+image:
+  repository: nitnelave/lldap
+  pullPolicy: IfNotPresent
+  tag: latest-debian
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+
+podSecurityContext:
+  runAsUser: 568
+  runAsGroup: 568
+  fsGroup: 568
```

Comment:
This is not new-common compatible specifically.

Reply:
Removed.